docs/designs/sqlflow_integration.md

```diff
@@ -65,7 +65,7 @@ PREDICT prediction_results_table
 WITH
     model.num_classes = 10,
     model.dataset_fn = "dataset_fn",
-    predict.checkpoint_filename_for_init = "modelv1.chkpt",
+    predict.checkpoint_dir_for_init = "v1/",
```

Comment:
No need to add "/" in the string?

Reply:
Yes, "/" is not needed. I want to show "v1/" is a directory not a file by adding "/".